setuptools/command/build_py.py

```diff
@@ -90,10 +89,7 @@ def __getattr__(self, attr: str):
         return orig.build_py.__getattr__(self, attr)
 
     def build_module(self, module, module_file, package):
-        outfile, copied = orig.build_py.build_module(self, module, module_file, package)
-        if copied:
-            self.__updated_files.append(outfile)
```

Comment:
Looks like `__updated_files` was completely unused, and its mangled

Reply:
I see, apparently it was introduced for the `2to3` transformation: https://github.com/pypa/setuptools/commit/5b568886e649fad8811f81484dc88fa5111faa94.

In this case, does it make sense to keep this method? (It will only call `super()` right?)

Reply:
You're right! I didn't even notice that. I'd remove the method entirely at this point.

Reply:
I agree, it is better to remove it.